core/domain/question_domain.py

```diff
@@ -0,0 +1,145 @@
+# coding: utf-8
+#
+# Copyright 2017 The Oppia Authors. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS-IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+"""Domain objects relating to question."""
```

Comment:
question --> questions

Reply:
Updated.

Reply:
@giritheja -- I'm looking again at this PR, and I don't see the PR changing in response to some old comments. Are you sure you've addressed everything?

Reply:
@giritheja This comment still isn't addressed.